flash_examples/integrations/learn2learn/image_classification_imagenette_mini.py

```diff
@@ -14,71 +14,139 @@
 
 # adapted from https://github.com/learnables/learn2learn/blob/master/examples/vision/protonet_miniimagenet.py#L154
 
+"""
+Requirements:
+
+pip install learn2learn
+pip install kornia
+pip install lightning-flash
+pip install 'lightning-flash[image]'
+"""
+
```

Comment:
Thanks for adding this here. I understand the motivation for adding this, but let's remove it as we expect users to install flash, kornia and learn2learn if they are using it. If they are running this script from the flash repository, they can do `pip install -r requirements/datatype_image_extras.txt".

```suggestion
```